makefiles/features_modules.inc.mk

```diff
@@ -28,3 +28,8 @@ endif
 
 # select cortexm_svc pseudomodule if the corresponding feature is used
 USEMODULE += $(filter cortexm_svc, $(FEATURES_USED))
+
+# use mpu_stack_guard if the feature is used
+ifneq (,$(filter cortexm_mpu,$(FEATURES_USED)))
+  USEMODULE += mpu_stack_guard
+endif
```

Comment:
enabling in `mpu_stack_guard` when `cortexm_mpu` is included feels like the reverse order of things, but I guess this is a limitation of the current build system.

Reply:
agreed... there should be optional dependencies, as in, "use mpu_stack_guard if it and all its deps are available".

Reply:
Ok, then let's not block on this build system issue